Fix wording for 'CREATE CACHE ALWAYS'

lukoktonos opened the pull request:
The things that get executed against readyset with `ALWAYS` are queries, not `CREATE CACHE` commands themselves.

jseldess reviewed (approved):
LGTM. Thanks again, @KianNH!

Commit:
Fix wording for 'CREATE CACHE ALWAYS'

The things that get executed against readyset with `ALWAYS` are queries,
not `CREATE CACHE` commands themselves.

zoready reviewed (approved):
LGTM
